corehq/apps/accounting/views.py

```diff
@@ -442,7 +442,7 @@ def invoice_context(self):
 
         if self.subscription.account.is_customer_billing_account:
             invoice_report = CustomerInvoiceInterface(self.request)
-            invoice_report.filter_by_account(self.subscription.account)
+            invoice_report.filter_by_subscription(self.subscription)
```

Comment:
This changes the report from filtering based on `account` to filtering based on `subscription`. Is that intentional, and if so, can you explain what affects that change will have? It seemed like that choice was intentional since that was the main distinguishing feature between the two branches of the `if` statement, and this section is specifically for enterprise billing accounts.

Reply:
Yes, it's intentional. The affect is, after user click on "edit" for that subscription, in the invoice tab, we will only return all customer invoices including that subscription. (A customer invoice usually is an aggregation of many subscriptions under that customer billing account)

Reply:
Thanks Biyeun! Fixed in c8bb0c177775ab5893a222e7f4e95c76011cf835 and 36f2937b78435113f09beb33652e3c82392ce3df